docs/extend/plugins_volume.md

```diff
@@ -12,10 +12,10 @@ parent = "mn_extend"
 
 Docker volume plugins enable Docker deployments to be integrated with external
 storage systems, such as Amazon EBS, and enable data volumes to persist beyond
-the lifetime of a single Docker host. See the [plugin documentation](plugins.md)
+the lifetime of a single Docker host. See the [plugin documentation](plugin_api.md)
```

Comment:
HI @thaJeztah I'd rather link them to the top rather than to the API directly.  The top will give them an overview.  The API not so much.